src/pages/clients-list/forms/ClientTemplateForm.js

```diff
@@ -738,7 +738,7 @@ const ClientTemplateForm = ({ recordId, labels, plantId, maxAccess, allowEdit =
           <Grid container xs={12} spacing={2} sx={{ padding: '20px' }}>
             <Grid item xs={6} sx={{ padding: '30px' }}>
```

Comment:
there is no need for the padding here the spacing from the parents should be enough